lib/screens/home/home_header_widget.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:survey_flutter/gen/assets.gen.dart';
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/theme/app_constants.dart';
+import 'package:intl/intl.dart';
+import 'package:survey_flutter/utils/build_context_ext.dart';
+
+class HomeHeaderWidget extends StatelessWidget {
+  const HomeHeaderWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final currentDateFormatted = _formatCurrentDate();
+
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: Metrics.spacing20,
+          vertical: Metrics.spacing28,
+        ),
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            _buildHeaderInfo(context, currentDateFormatted),
+            _buildProfileAvatar(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  String _formatCurrentDate() {
+    final now = DateTime.now();
+    return DateFormat('EEEE, MMMM dd').format(now).toUpperCase();
+  }
+
+  Widget _buildHeaderInfo(BuildContext context, String currentDateFormatted) {
+    return Expanded(
```

Comment:
We should not use `Expaned` as a returned value, since it will limit the reusable. Let the parent decide to use Expaned or not 🙏